ee/session_recordings/ai/similar_recordings.py

```diff
@@ -46,7 +46,7 @@ def closest_embeddings(session_id: str, team_id: int):
                 AND generation_timestamp > now() - INTERVAL 7 DAY
                 -- skip the target recording
                 AND session_id != %(session_id)s
-            ORDER BY similarity_score DESC
+            ORDER BY similarity_score ASC
```

Comment:
Just a nit, you can omit `asc` since it's the default sorting, but also fine to keep it explicit.